### PR TITLE
created models, migrations, and controllers

### PR DESCRIPTION
**Pull request by stockdalemark1978**

### CODE REVIEW

**Comment by doubldragon** [comment]
delayed... +.35

File: ibackend/app/EquipmentType.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class EquipmentType extends Model
+{
+    //
```
Comment (doubldragon) [inline ibackend/app/EquipmentType.php]: Should this have the "hasMany" for equipment

File: ibackend/app/Role.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Role extends Model
+{
+    //
```
Comment (doubldragon) [inline ibackend/app/Role.php:9]: "hasMany" users

File: ibackend/app/User.php [hunk 3]
```diff
@@ -9,6 +9,9 @@ class User extends Authenticatable
 {
     use Notifiable;
 
+    // Enable soft deletes for this class
+    use SoftDeletes;
+
     /**
      * The attributes that are mass assignable.
```
Comment (doubldragon) [inline ibackend/app/User.php:16]: "belongsTo" role

**Comment by doubldragon** [comment]
+1

**Review by Brayn7 — APPROVED** [review]
+1

**Pull request merged**